svelte.config.js

```diff
@@ -1,29 +1,47 @@
-import adapter from "@sveltejs/adapter-auto";
+import path from 'path';
+import adapter from "@sveltejs/adapter-vercel";
 import preprocess from "svelte-preprocess";
+import {mdsvex} from 'mdsvex';
+import slug from 'rehype-slug';
+import Prism from 'prismjs';
+import 'prism-svelte';
+
+const dev = process.env.NODE_ENV === 'development';
 
 /** @type {import('@sveltejs/kit').Config} */
 const config = {
-  // Consult https://github.com/sveltejs/svelte-preprocess
-  // for more information about preprocessors
+  extensions: ['.svelte', '.md'],
   preprocess: [
     preprocess({
       postcss: true,
     }),
+    mdsvex({
+      extensions: ['.md'],
+      rehypePlugins: [slug],
+      highlight: function (str, lang) {
+        if (lang && lang in Prism.languages) {
+          try {
+            return Prism.highlight(str, Prism.languages[lang], lang);
+          } catch(__) {}
+        }
+        return '';
```

Comment:
It's preferable to abstract this logic into the mdxvex config file to keep the svelte config focused and easy to grok.